tests/components/discord/test_config_flow.py

```diff
@@ -147,7 +147,7 @@ async def test_flow_import_no_name(hass: HomeAssistant) -> None:
 
     assert result["type"] == data_entry_flow.RESULT_TYPE_CREATE_ENTRY
     assert result["title"] == NAME
-    assert result["data"] == CONF_DATA
+    assert result["data"] == CONF_DATA | {CONF_NAME: DEFAULT_NAME}
```

Comment:
This makes a non deterministic test. I think it should split into two test (or two sub-tests in the same test) so that the assert will be deterministic, otherwise if someone will remove or change the `DEFAULT_NAME` or break the `CONF_DATA` it will still pass.

Reply:
I am rethinking this PR. Instead of conforming the service to be `notify.discord` for import, the documentation should make it clear that the service name will match what was set up in the Discord dev portal.

Reply:
I think people are looking at the example service call and assuming they should `notify.discord` listed.